### ISSUE TITLE
What has the model done during a warm-up time?

### ISSUE THREAD

**Issue by erow** [snu-mllab/DisentanglementICML19#2]
```python
  model = Model(dm, LOG_DIR+FILE_ID+'.log', args)
  model.set_up_train()
  model.initialize()
  model.train(niter=NITER, siter=SITER, piter=PITER, save_dir=SAVE_SUBDIR, asset_dir=ASSET_SUBDIR)
  model.restore(save_dir=SAVE_SUBDIR)
```
I am not familiar with tensorflow, but it seems the model only initiates its params during the warm-up time. However, the algorithm in your paper shows that the model updates the params through the whole training process. It would be strange if the warm-up process only initiates some params.
![image](https://user-images.githubusercontent.com/7524356/112273222-0f177a00-8cb8-11eb-81e5-835419f421df.png)
